numba/np/is_comparison_tests.py

```diff
@@ -0,0 +1,75 @@
+import numpy as np
+from arrayobj import test_kernel
+
+def test_structured_is():
+    a = np.zeros(1, dtype=[("a", "i4"), ("b", "i4")])[0]
+    b = a.copy()
+    test_kernel(a, b)
+
+def test_cuda_structured_is():
+    a = np.zeros(1, dtype=[("a", "i4"), ("b", "i4")])[0]
+    b = a.copy()
+    cuda.jit(test_kernel)[1, 1](a, b)
+    cuda.synchronize()
+
+def test_record_identity():
+    a = np.zeros(1, dtype=[("a", "i4"), ("b", "i4")])[0]
+    b = a.copy()
+
+    result = test_kernel(a, b)
```

Comment:
`test_kernel` returns `None` in its present implementation (which needs removing from `numba.np.arrayobj` also).